ursh/indico_ursh/views.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask_pluginengine import render_plugin_template
+
+from indico.core.plugins import WPJinjaMixinPlugin
+from indico.web.views import WPDecorated
+
+
+class WPShortenURLPage(WPJinjaMixinPlugin, WPDecorated):
+
+    def _getBody(self, params):
+        return render_plugin_template('url_shortener.html', **params)
```

Comment:
check in other places where we render templates from plugins; we usually call `WPSomething.render_template('foo.html')` instead of manually instantiating the WP and calling display (that's kind of legacy)